extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -114,6 +114,21 @@ export class AmpAutocomplete extends AMP.BaseElement {
      */
     this.container_ = null;
 
+    /**
+     * The reference to the <form> ancestor that contains <amp-autocomplete>.
+     * @private {?Element}
+     */
+    this.formAncestor_ = null;
+
+    /**
+     * The developer specified value of the 'autocomplete' attribute on the
+     * <form> ancestor that contains <amp-autocomplete>. Used to reset the
+     * attribute on blurring the input field. 'on' by default, according to
+     * common browser practices.
+     * @private {string}
+     */
+    this.formAncestorAutocompleteAttr_ = 'on';
```

Comment:
Could you add a link to the spec stating that `on` is the default value? I wasn't certain and had to look it up. A top-level `const` variable could also help the code self-document


> If the autocomplete attribute is omitted, the default value corresponding to the state of the element’s form owner’s autocomplete attribute is used instead (either "on" or "off"). If there is no form owner, then the value "on" is used.
>
> https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute

Reply:
Since I've modified the code to either reset the attribute to its initial value or remove it, I've removed this entirely.